docker/addons/cassandra-reader/docker-compose.yml

```diff
@@ -23,8 +23,12 @@ services:
       MF_CASSANDRA_READER_PORT: ${MF_CASSANDRA_READER_PORT}
       MF_CASSANDRA_READER_DB_CLUSTER: ${MF_CASSANDRA_READER_DB_CLUSTER}
       MF_CASSANDRA_READER_DB_KEYSPACE: ${MF_CASSANDRA_READER_DB_KEYSPACE}
+      MF_CASSANDRA_READER_SERVER_CERT: ${MF_CASSANDRA_READER_SERVER_CERT}
+      MF_CASSANDRA_READER_SERVER_KEY: ${MF_CASSANDRA_READER_SERVER_KEY}
```

Comment:
same as other variables shoud be put here https://github.com/mainflux/mainflux/blob/master/.env#L128